cdap-ui/app/cdap/services/WizardStores/AddNamespace/ActionCreator.js

```diff
@@ -16,7 +16,15 @@
 import AddNamespaceStore from 'services/WizardStores/AddNamespace/AddNamespaceStore';
 import {MyNamespaceApi} from 'api/namespace';
 
-const PublishNamespace = () => {
+const publishNamespace = () => {
```

Comment:
nit: May be just `saveNamespace`? and `savePreferences`? Don't know if we could say 'publish namespace' here.